src/System.Diagnostics.Tools/pkg/any/System.Diagnostics.Tools.pkgproj

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <!-- there is no "jit" rid, so we use any and "aot" will override -->
+    <PackageTargetRuntime>any</PackageTargetRuntime>
+    <PreventImplementationReference>true</PreventImplementationReference>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\System.Diagnostics.Tools.csproj" />
+
+    <!-- AOT implementation comes from AOT package -->
```

Comment:
It makes it so that if someone directly installs the any package it won't contribute assets on aot.  Its a bit of redundancy, safeguarding the direct install.